.github/workflows/ruby.yml

```diff
@@ -19,12 +19,19 @@ jobs:
         experimental: [false]
         additional_engine_cart_rails_options: [""]
         additional_name: [""]
+        view_component_version: ["~> 3.12"]
         include:
           - ruby: "3.1"
             rails_version: 6.1.7
             blacklight_version: "~> 7.0"
             experimental: false
             additional_name: Rails 6.1, Ruby 3.1
+          - ruby: "3.1"
+            rails_version: 6.1.7
```

Comment:
Why add this with a deprecated version of rails  and ruby?

Reply:
Right now I am just trying to get ViewComponent2 to install, which isn't working.